### PR TITLE
Awesome books: with ES6

### PR DESCRIPTION
**Pull request by kambereBr**
In this MR, the features below has been implemented :
 - A Navigation bar.
  - Three content sections:
    1. Books list.
    2. Add book form.
    3. Contact info.

- The project has been implemented using: 

1. ES6 modules to write modular JavaScript.
2. Proper ES6 syntax.
3. npm as software packages system.

### CODE REVIEW

**Comment by omarsalem7** [comment]
Hi @kambereBr 😊, 

While you made a great effort in this project, I cannot proceed to review your code.

## Invalid Code Review Request 🛑 💀 

You have submitted a project, that doesn't have a descriptive readme, kindly update it and consider using the Microverse readme template

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by kambereBr** [comment]
Thanks @omarsalem7 for your feedback

File: assets/script/modules/functions.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { DateTime } from '../../../node_modules/luxon/src/luxon.js';
+
+function myTimer() {
+  const date = DateTime.now().toLocaleString(DateTime.DATETIME_FULL_WITH_SECONDS);
+  document.querySelector('.date').innerHTML = date;
+}
```
Comment (fobadara) [inline assets/script/modules/functions.js]: [REQUIRED] Kindly use arrow function as this is a requirement for this milestone

File: assets/script/modules/navigation-menu.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  const listMenu = document.getElementById('list-menu');
+  const addMenu = document.getElementById('add-menu');
+  const contactMenu = document.getElementById('contact-menu');
+
+  if (id === 'list-menu') {
+    bookList.style.display = 'block';
+    bookCreation.style.display = 'none';
+    contact.style.display = 'none';
+    listMenu.classList.toggle('active');
+    addMenu.classList.remove('active');
```
Comment (fobadara) [inline assets/script/modules/navigation-menu.js]: [REQUIRED] Kindly use arrow function as this is a requirement for this milestone

File: assets/script/modules/navigation-menu.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
+/* SPA manipulation section */
+function loadContent(id) {
+  const bookList = document.querySelector('.book-item-wrapper');
+  const bookCreation = document.querySelector('.addbook-section');
+  const contact = document.querySelector('.contact-section');
+
+  const listMenu = document.getElementById('list-menu');
+  const addMenu = document.getElementById('add-menu');
+  const contactMenu = document.getElementById('contact-menu');
+
```
Comment (fobadara) [inline assets/script/modules/navigation-menu.js]: [REQUIRED] Kindly use arrow function as this is a requirement for this milestone

**Review by fobadara — CHANGES_REQUESTED** [review]
Hi @kambereBr, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## Highlights
✅ Good use of modules for separation of concerns
✅ Descriptive README

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: assets/script/modules/functions.js (hunk 1 above)
Comment (kambereBr) [reply assets/script/modules/functions.js]: @fobadara thanks for your feedback

**Review by fobadara — APPROVED** [review]
Hi @kambereBr, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✅ Good use of modules
✅ Clean code

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**